attribute-masking-service/src/component-tests/java/uk/gov/gchq/palisade/component/attributemask/model/AttributeMaskingRequestTest.java

```diff
@@ -15,90 +15,58 @@
  */
 package uk.gov.gchq.palisade.component.attributemask.model;
 
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
 import org.junit.jupiter.api.Test;
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.boot.test.autoconfigure.json.JsonTest;
-import org.springframework.boot.test.json.JacksonTester;
-import org.springframework.boot.test.json.JsonContent;
-import org.springframework.boot.test.json.ObjectContent;
-import org.springframework.test.context.ContextConfiguration;
 
-import uk.gov.gchq.palisade.Context;
-import uk.gov.gchq.palisade.User;
-import uk.gov.gchq.palisade.resource.LeafResource;
-import uk.gov.gchq.palisade.resource.impl.FileResource;
-import uk.gov.gchq.palisade.resource.impl.SystemResource;
-import uk.gov.gchq.palisade.rule.Rules;
-import uk.gov.gchq.palisade.service.SimpleConnectionDetail;
+import uk.gov.gchq.palisade.service.attributemask.common.Context;
```

Comment:
The package name common, is this an intermediate step in the refactoring or the final destination?  Some of these objects could be better suited for existing packages such as User could be moved to the model package.

Reply:
I think if they are external to the module, and were previously dependencies, they should remain in common.
If they change and become module specific then they can be pulled into each modules model package

Reply:
Sounds good